connector/servicegraphconnector/connector_test.go

```diff
@@ -143,16 +143,16 @@ func verifyCount(t *testing.T, m pmetric.Metric) {
 	verifyAttr(t, attributes, "client_some-attribute", "val")
 }
 
-func verifyDuration(t *testing.T, m pmetric.Metric, durationSum float64) {
+func verifyDuration(t *testing.T, m pmetric.Metric, durationSum float64, bucktes []uint64) {
 	assert.Equal(t, pmetric.MetricTypeHistogram, m.Type())
 	dps := m.Histogram().DataPoints()
 	assert.Equal(t, 1, dps.Len())
 
 	dp := dps.At(0)
-	assert.Equal(t, durationSum, dp.Sum()) // Duration: 1sec
+	assert.Equal(t, durationSum, dp.Sum()) // Duration: client is 1sec, server is 2sec
```

Comment:
Since `bucktes` is now passed from outside, the duration assert here is a common judgment, and I don't think a sub-comment is appropriate

Reply:
The previous UT did not detect any issues due to the server and client latency being set to 1 second, 
so if we want to discover some issues, we should keep the latency of the server and client different values.
In this UT test, I set the server's duration to 2 seconds and the client's duration to 1 second. It's best to have a comment here to explain